images/metering-ansible-operator/roles/meteringconfig/tasks/validate_reporting_operator_tls.yml

```diff
@@ -10,6 +10,17 @@
       that:
         - meteringconfig_spec['reporting-operator'].spec.config.presto.tls.caCertificate != ""
       msg: "reporting-operator.spec.config.presto.tls.caCertificate cannot be empty if createSecret: true and secretName != ''"
+  rescue:
```

Comment:
Ansible noob, so bear with me: Would it be possible to pull a lot of these tasks under one `block:` statement so that the `rescue:` could apply to all and not be repeated so much?

Reply:
In this case, that isn't something that needs to be reflected in the status, and only in the case of a failure (i.e. the `rescue` block) do we care about what happened.

In other cases, unfortunately not - it requires a lot more work to be able to track what's currently happening, and what went wrong (especially when looping over resources).